sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeBlockSuite.scala

```diff
@@ -117,7 +117,7 @@ class CodeBlockSuite extends SparkFunSuite {
 
   test("Throws exception when interpolating unexpected object in code block") {
     val obj = Tuple2(1, 1)
-    val e = intercept[IllegalArgumentException] {
+    val e = intercept[SparkException] {
       code"$obj"
     }
     assert(e.getMessage().contains(s"Can not interpolate ${obj.getClass.getName}"))
```

Comment:
As you are here, please, use `checkError` to check the error class and other parameters.